Scala/Scala.sublime-syntax

```diff
@@ -265,3 +349,24 @@ contexts:
           pop: true
         - include: xml-literal
         - include: xml-attribute
+  pattern-match:
+    - include: comments
+    - include: block-comments
+    - include: keywords
+    - include: constants
+    - include: char-literal
+    - include: scala-symbol
+    - include: strings
+    - include: xml-literal
+    - match: '`'
+      push:
+        - meta_scope: source.scala
+        - match: '`'
+          pop: true
+    - match: '\b[a-z][a-zA-Z0-9_]*\b'
```

Comment:
Yep. This falls into the same category as "all the other regexes that are slightly, but not completely, inconsistent and wrong".  Extracting everything into variables is the way to fix that, but I only want to do that once, rather than once per outstanding PR.